Populate about section of landing page

milesswu opened the pull request:
In accordance #3, created and populated the about section of our landing page.

milesswu committed:
Populate about section of landing page

In accordance #3, created and populated the about section of our landing page.

risesummer commented on index.html:
```diff
@@ -5,5 +5,16 @@
+	<body>
+		<h1>About</h1>
+		<p>
+			Lorem ipsum dolor, sit amet consectetur adipisicing elit. Amet dolorum
```
why is this still placeholder text??

milesswu committed:
Replace placeholder text

In accordance with @risesummer, modified about section to no longer use
placeholder text.
